src/lava/lib/dl/netx/blocks/process.py

```diff
@@ -0,0 +1,237 @@
+                bias_mant = bias
+            elif list(bias.shape) == list(self.shape):
+                # neuron wise bias value
+                bias_mant = bias
+            elif self.shape[-1] == bias.shape[0]:  # channel wise bias value
+                bias_mant = np.zeros(self.shape)
+                bias_mant[:, :] = bias
+            else:
+                raise RuntimeError(
+                    f'Bias of shape {bias.shape} could not be broadcast to '
+                    f'neuron of shape {self.shape}.'
+                )
+            # print(f'{bias=}')
+            # print(f'{bias_mant=}')
+            neuron_params = {'shape': self.shape, **self.neuron_params}
+            neuron_params['bias'] = bias_mant.astype(np.int32)
+            return self.neuron_process(**neuron_params)
+
+    @property
+    def block(self) -> str:
+        return str(self.__class__.__name__)
+
+
+class Input(AbstractBlock):
+    """Input layer block.
+
+    Parameters
+    ----------
+    shape : tuple or list
+        shape of the layer block in (x, y, z)/WHC format.
+    neuron_params : dict
+        dictionary of neuron parameters. Defaults to None.
+    transform : fx pointer or lambda
+        input transform to be applied. Defualts to ``lambda x: x``.
+    bias : np.ndarray or None
+        bias of input neuron. None means no bias. Defaults to None.
+    """
+    def __init__(self, **kwargs: Union[dict, tuple, list, int, bool]) -> None:
+        super().__init__(**kwargs)
+        self.transform = kwargs.pop('transform', lambda x: x)
+
+        if 'bias_exp' not in self.neuron_params:
+            self.neuron_params['bias_exp'] = 6
+
+        self.neuron = self._neuron(kwargs.pop('bias', None))
+
+        # The input must be handled through neuron's bias.
+        self.inp = InPort(shape=self.neuron.a_in.shape)
+        self.out = OutPort(shape=self.neuron.s_out.shape)
+        self.neuron.s_out.connect(self.out)
+
+        self._clean()
+
+    def export_hdf5(self, handle: Union[h5py.File, h5py.Group]) -> None:
+        raise NotImplementedError
+
+
+class Dense(AbstractBlock):
+    """Dense layer block.
+
+    Parameters
+    ----------
+    shape : tuple or list
+        shape of the layer block in (x, y, z)/WHC format.
+    neuron_params : dict
+        dictionary of neuron parameters. Defaults to None.
+    weight : np.ndarray
+        synaptic weight.
+    bias : np.ndarray or None
+        bias of neuron. None means no bias. Defaults to None.
+    has_graded_input : dict
+        flag for graded spikes at input. Defaults to False.
+    """
+    def __init__(self, **kwargs: Union[dict, tuple, list, int, bool]) -> None:
+        super().__init__(**kwargs)
+
+        weight = kwargs.pop('weight')
+        num_weight_bits = kwargs.pop('num_weight_bits', 8)
+        weight_exponent = kwargs.pop('weight_exponent', 0)
+        sign_mode = kwargs.pop('sign_mode', 1)
+
+        graded_spikes_params = {'use_graded_spike': self.has_graded_input}
```

Comment:
Seems like a mismatch of docstring params to init attributes set. Perhaps update?

Reply:
The params are right. They get initialized in the Abstract class constructor. I forgot to add params for num_delay_bits and weight_exponent. Added them.